### DISCUSSION TITLE
How to add a style only to one specific column once it's sorted

### DISCUSSION THREAD

**Discussion by ghost** [TanStack/table#3651] [General]
Hello.

I want to add margin-left only to one specific column of the table once its header is clicked. 
Right now it works that once I click on <StyledTitleAndSortedIconWrapper> in one column header each header in the table gets the same margin. Logic is executed by flag "isSorted". Can You please advise how to achieve the result where margin will be added only to column which is clicked ? 

` const [isSorted, setIsSorted] = useState(false);

    const RenderDesktopVersion = ({ column, idx }) => {
        return (
            <StyledHeaderWrapperDesktop>
                <StyledTitleAndSortedIconWrapper
                    onClick={() =>
                        !column.isSortedDesc
                            ? setIsSorted('true')
                            : setIsSorted('false')
                    }
                >
                    <span
                        {...column.getHeaderProps(
                            column.getSortByToggleProps(),
                            {
                                className: column.collapse ? 'collapse' : '',
                            }
                        )}
                    >
                        {column.render('Header')}
                    </span>
                    <span>
                        {column.isSorted ? (
                            column.isSortedDesc ? (
                                <StyledArrowDownwardIcon />
                            ) : (
                                <StyledArrowUpwardIcon />
                            )
                        ) : (
                            ''
                        )}
                    </span>
                </StyledTitleAndSortedIconWrapper>
                <span
                    style={{
                        marginLeft: isSorted ? '12px' : '0px',
                    }}
                >
                    {columnsCount > 5 ? (
                        <StyledHeaderDivFlex>
                            {column.canFilter ? column.render('Filter') : null}
                        </StyledHeaderDivFlex>
                    ) : (
                        <StyledHeaderDivBlock>
                            {column.canFilter ? column.render('Filter') : null}
                        </StyledHeaderDivBlock>
                    )}
                </span>
            </StyledHeaderWrapperDesktop>
        );
    };`

thanks in advance for support

**Comment by Hasham-dev** [comment]
@marcinb1986 Can you share this code example?
